UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/RnwService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Microsoft.Extensions.Options;
+using UKHO.MaritimeSafetyInformation.Common.Configuration;
+using UKHO.MaritimeSafetyInformation.Common.Logging;
+using UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning;
+using UKHO.MaritimeSafetyInformation.Web.Services.Interfaces;
+
+namespace UKHO.MaritimeSafetyInformation.Web.Services
+{
+    public class RnwService : IRnwService
+    {
+        private readonly IRnwRepository _rnwRepository;
+        private readonly IOptions<RadioNavigationalWarningConfiguration> _radioNavigationalWarningConfiguration;
+        private readonly ILogger<RnwService> _logger;
+        private static List<RadioNavigationalWarningsAdminList> s_allRadioNavigationalWarningsAdminList;
+
+        public RnwService(IRnwRepository repository,
+                        IOptions<RadioNavigationalWarningConfiguration> radioNavigationalWarningConfiguration,
+                        ILogger<RnwService> logger)
+        {
+            _rnwRepository = repository;
+            _radioNavigationalWarningConfiguration = radioNavigationalWarningConfiguration;
+            _logger = logger;
+        }
+
+        public async Task<RadioNavigationalWarningsAdminListFilter> GetRadioNavigationWarningsForAdmin(int pageIndex, int warningType, string year, bool reLoadData, string correlationId)
+        {
```

Comment:
Please add unit test for RnwService class.

Reply:
Added unit test for RnwService class.